components/support/base/src/main/java/mozilla/components/support/base/crash/Breadcrumb.kt

```diff
@@ -103,11 +103,7 @@ data class Breadcrumb(
     }
 
     override fun compareTo(other: Breadcrumb): Int {
-        if (this.level.ordinal == other.level.ordinal) {
-            return this.date.compareTo(other.date)
-        }
-
-        return this.level.ordinal.compareTo(other.level.ordinal)
+        return this.date.compareTo(other.date)
```

Comment:
Alternatively we could stop using a `PriorityQueue` and add at the end and remove items over the limit from the front?

Reply:
That's true.  Originally we did this because we want to sort by priority.  But we don't need that anymore.  I'll update with a simpler implementation.